src/types/todoCompletedCategory.ts

```diff
@@ -0,0 +1,5 @@
+export enum TodoCompletedCategory {
+  all = 'all',
+  active = 'active',
+  competed = 'completed',
```

Comment:
There is a typo in the enum value `competed`. It should be `completed` to accurately represent the category for completed todos.